dev/doc/critical-bugs

```diff
@@ -2,8 +2,78 @@ Preliminary compilation of critical bugs in stable releases of Coq
 ==================================================================
   WORK IN PROGRESS WITH SEVERAL OPEN QUESTIONS
 
+##################################################################
+##################### Non fixed bugs #############################
+##################################################################
 
-To add: #7723 (vm_compute universe polymorphism), #7695 (modules and algebraic universes), #7615 (lost functor substitutions)
+  component: VM reduction machine
+  summary: Memory corruption
+  introduced: 8.1
+  impacted versions: 8.1-NOW
+  impacted coqchk versions: none (no VM in coqchk)
+  fixed in: NONE
+  found by: Dolan, Roux, Melquiond
+  GH issue number: #13439
+  exploit: ??
+  risk: requires very large number of arguments, fix block size or nested letins
+
+  component: VM and native reduction machines
+  summary: memory corruption by evaluating on ill-typed terms (obtained from unsafe tactics)
+  introduced: 8.1
+  impacted versions: 8.1-NOW
+  impacted coqchk versions: none (no VM or native in coqchk)
+  fixed in: NONE
+  found by: Gaëtan Gilbert, Andres Erbsen
+  GH issue number: #16891
+  exploit: requires a memory corruption to craft something that doesn't just SIGSEV
+  risk: could be activated by chance but unlikely to produce anything
+        other than SIGSEV outside a deliberate attack
+
+  component: name handling / typechecker
+  summary: kernel and checker accept incorrect name aliasing information
+  introduced: a long time ago
+  impacted versions: -NOW
+  impacted coqchk versions: same
+  fixed in: NONE
+  found by: Pierre-Marie Pédrot
+  GH issue number: #7609
+  exploit: see issue (requires a plugin or hand crafted .vo file)
+  risk: low
+
+  component: primitive types and operators
+  summary: coqchk checks too little about primitive declarations
+  introduced: v8.10 (#6914 primitive integers)
+  impacted versions: coqchk only
+  impacted coqchk versions: V8.10-NOW
+  fixed in: NONE
+  found by: Gaëtan Gilbert
+  GH issue number: #12439
+  exploit: not fully worked out, requires crafted .vo file
+  risk: low
```

Comment:
I would classify it as "risk: none, as it requires crafted .vo files". I consider that our risk assessment should only apply to non-adversarial users, that is, what is the chance that a standard user triggers the bug inadvertently. (When the user is adversarial, the risk is always "critical" anyway.)